kitsune/messages/jinja2/messages/outbox.html

```diff
@@ -75,7 +74,10 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
                       {% if message.to_group %}
                         {% if message.to_groups_count > 0 -%}
                           {%- for group in message.to_group.all()[:3] -%}
-                            {{ group_link(group) }}
+                            {%- for profile in group.profile.all() -%}
```

Comment:
this feels too expensive for a big group, why are we adding this?

Reply:
`group_link(group)` is a global jinja helper, but it makes a db hit for every group passed to it. This change takes advantage of having prefetched the `to_group` in the view. I didn't want to change the global helper function since it is also used in the `users` app.